kayvee.go

```diff
@@ -19,7 +20,11 @@ const (
 
 // Format converts a map to a string of space-delimited key=val pairs
 func Format(data map[string]interface{}) string {
-	formattedString, _ := json.Marshal(data)
+	formattedString, err := json.Marshal(data)
+	if err != nil {
+		log.Printf("Error converting kayvee message to json: %s", err.Error())
```

Comment:
I thought about that, but wasn't sure what to do. We probably shouldn't have it call `Format` recursively right? That seems weird. Should I just call json.Marshal inside the error case with something we know will be valid?